README.md

```diff
@@ -21,17 +21,23 @@ with the docker-compose.yml from this repository.
 
 ### Prerequisites
 
-For frontend:
+#### For frontend
 
 - [Node.js](https://nodejs.org/en/download) (includes npm)
 - [Mockoon](https://mockoon.com/download/#download-section)
 
-For backend
+#### For backend
 
 - [Python 3.12](https://www.python.org/downloads/)
 - [UV](https://docs.astral.sh/uv/getting-started/installation/#standalone-installer)
 - [Docker](https://www.docker.com/get-started/)
 
+On Linux you must instal the mysql dev binaries to be able to build the `mysqlclient` python distribution.
+
+- Debian / Ubuntu: `apt-get install default-libmysqlclient-dev`
+- Red Hat / CentOS: `yum install mysql-devel`
+- Arch: `pacman -S mysql`
+
```

Comment:
Just as additional information, I recently learned there's a proposal to solve this need: https://peps.python.org/pep-0725/